docs/sources/static/release-notes.md

```diff
@@ -28,6 +28,16 @@ For a complete list of changes to Grafana Agent, with links to pull requests and
 [release-notes-flow]: "/docs/grafana-cloud/ -> ../flow/release-notes"
 {{% /docs/reference %}}
 
+## v0.36
```

Comment:
We cut the RC already, so this will need an update.

Reply:
so v0.37?

Reply:
yes